### DISCUSSION TITLE
'reanimated/apple/REASlowAnimations.h' file not found

### DISCUSSION THREAD

**Discussion by mcervantes3** [software-mansion/react-native-reanimated#6615] [Q&A]
Getting this error when I try to build my app in xCode:

` 'reanimated/apple/REASlowAnimations.h' file not found
`

I have cleaned the build folder, rebuilt, cleared the cache, restart xcode, restart the project, re-installed the package, re-cloned the project, even shut down the computer and restarted it

react-native: 0.73.0
react-native-animated: "^3.15.4"
xcode version: 16.0
macos: Sonoma 14.7

REASlowAnimations.h is in the project:

`namespace reanimated {

CGFloat getUIAnimationDragCoefficient(void);
CFTimeInterval calculateTimestampWithSlowAnimations(CFTimeInterval currentTimestamp);

} // namespace reanimated`

**Comment by MatiPl01** [comment]
Hey @mcervantes3!

Have you managed to solve the problem? I haven't seen any similar issue before. It looks like pods weren't installed properly but it's hard to say why.

If you still have a problem, would you be able to share the minimal repro? It would be easier for me to debug on my side.
